diff -rupN a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt    2026-01-07 07:57:50.000000000 +0100
+++ b/CMakeLists.txt    2026-02-06 16:52:59.927109936 +0100
@@ -45,7 +45,7 @@ if(NOT "$ENV{CURL_BUILDINFO}$ENV{CURL_CI
   endforeach()
 endif()

-set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/CMake" ${CMAKE_MODULE_PATH})
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/CMake")
 include(Utilities)
 include(Macros)
 include(CMakeDependentOption)
